rai_test_utils/rai_test_utils/datasets/vision/object_detection_data_utils.py

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+import os
+import urllib.request as request_file
+import xml.etree.ElementTree as ET
+from zipfile import ZipFile
+
+import pandas as pd
+
+
+def load_fridge_object_detection_dataset_labels():
+    """Loads the labels for the fridge object detection dataset.
+
+    return: list of labels
+    rtype: list
+    """
+
+    src_images = "./data/odFridgeObjects/"
+
+    # Path to the annotations
+    annotations_folder = os.path.join(src_images, "annotations")
+
+    labels = []
+    label_dict = {'can': 1, 'carton': 2, 'milk_bottle': 3, 'water_bottle': 4}
+
+    # Read each annotation
+    for _, filename in enumerate(os.listdir(annotations_folder)):
+        if filename.endswith(".xml"):
+            print("Parsing " + os.path.join(src_images, filename))
+
+            root = ET.parse(
+                os.path.join(annotations_folder, filename)
+            ).getroot()
+
+            # use if needed
+            # width = int(root.find("size/width").text)
```

Comment:
How do I know if it's needed?

Reply:
Lines 50-53 mention it, in case the coordinates need to be normalized.